src/sidebar/components/tag-editor.js

```diff
@@ -207,17 +224,38 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
           );
         })}
       </ul>
-      <input
-        list={datalistId}
-        onInput={handleOnInput}
-        onKeyPress={handleKeyPress}
-        onKeyUp={handleKeyUp}
-        ref={inputEl}
-        placeholder="Add tags..."
-        className="tag-editor__input"
-        type="text"
-      />
-      {suggestionsList()}
+      <span
+        id={tagEditorId}
+        className="tag-editor__combobox-wrapper"
+        ref={closeWrapperRef}
+        role="combobox"
+        aria-controls={`${tagEditorId}-datalist`}
+        aria-expanded={suggestionsListOpen.toString()}
+        aria-owns={`${tagEditorId}-datalist`}
+        aria-haspopup="true"
```

Comment:
The value of `aria-haspopup` here should match the role of the popup container (`"list"` in this case) in ARIA 1.1. See https://www.w3.org/TR/wai-aria-1.1/#aria-haspopup. The value "true" is treated as if it were "menu" for backwards compatibility.

Reply:
The role of the popup container is "listbox" I'm assuming that is what you mean. I'll make that change